### PR TITLE
SPAAS-647 Only minify browser bundle; remove sourcemaps from non-minifed bundles

### PR DESCRIPTION
**Pull request by samisayegh**
**Background**

I am trying to bundle the coveo.analytics package inside IPX. However, I do not want Chrome to show this warning related to source maps:

source-map_warning

**Problem**

By default, webpack tries to pull the file specified in the package.json `browser` field. This file should be minified and have a source map, since it is also served by CDN directly into browsers.

It is possible to configure webpack to use a different field such as `main` though. However, the file specified in the coveo.analytics `main` is also minified. This is not typical among open-source libs.

As far as I know, the best practice is to not minify non-browser files because they will be consumed by a bundler. This leaves it up to the consumer as to whether they want to minify the file and/or create source maps.

**Solution**

I adjusted rollup to no longer minify libUMD. I also removed source maps for this output, but also the `module` and `react-native` outputs for consistency.

Unrelated to the core issue, I adjusted the vscode configuration to show the `dist` folder, and removed a rollup plugin that was not being used.

### CODE REVIEW

**Review by dlafreniere — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "scripts": {
         "lint:check": "prettier --check .",
         "lint:fix": "prettier --write .",
-        "build": "rollup -c",
+        "build": "npm run clean && rollup -c",
```
Comment (francoislg) [inline package.json:12]: Why was this needed? I'd rather not delete anything on "build"

**Review by francoislg — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (samisayegh) [reply package.json:12]: I thought my changes to rollup were not working because I could still see the `.js.map` source map files in the `dist` folder. I then realized that this was because the directory was not emptied from the files generated by the last run before the changes.

I feel cleaning before each build helps avoid such scenarios, but if you prefer to not do this, I will revert this change.

Comment (francoislg) [reply package.json:12]: `clean` cleans up a bit more things than the build, which is the main reason I am not super fond of it :sweat: 

I think we rarely **need** a clean state so the implicit delete does not look required to me

Comment (samisayegh) [reply package.json:12]: Ok, I will revert the change 👍
